src/main/java/com/f1soft/bachaat/entity/Menu.java

```diff
@@ -0,0 +1,83 @@
+package com.f1soft.bachaat.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "tbl_menu")
+public class Menu {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "menu_id")
+    private long id;
+    @NotNull(message = "Menu cannot be null.")
+    @Column(name = "menu_name")
+    private String name;
+    @Column(name = "parent_id")
+    private long parentId;
+    @NotNull(message = "Link cannot be null.")
+    private String link;
+    @Column(name = "menu_status")
```

Comment:
you can make simply status rather than menu_status

Reply:
Change to enable.